src/main/java/uk/gov/hmcts/divorce/citizen/notification/Applicant2RequestChangesNotification.java

```diff
@@ -88,11 +89,18 @@ public void sendToApplicant2(final CaseData caseData, final Long id) {
             );
         } else {
             log.info("Sending notification to applicant 2 to confirm their request for changes (applicant 1 is not represented): {}", id);
+            final var templateContent = commonContent.mainTemplateVars(caseData, id, caseData.getApplicant2(), caseData.getApplicant1());
+
+            LanguagePreference languagePreference = caseData.getApplicant2().getLanguagePreference();
+
+            if (LanguagePreference.WELSH.equals(languagePreference)) {
```

Comment:
Did you want to try the new CommonContent method here?

Reply:
yes will update all my PRs thanks